Code/autopkglib/MunkiImporter.py

```diff
@@ -539,23 +577,44 @@ def main(self):
             return
 
         # copy pkg/dmg to repo
-        relative_path = self.copy_item_to_repo(pkginfo)
+        uploaded_pkgpath = munkiimportlib.copy_item_to_repo(
+            repo,
+            self.env["pkg_path"],
+            pkginfo.get("version"),
+            self.env.get("repo_subdirectory", ""),
+        )
         # adjust the installer_item_location to match the actual location
         # and name
-        pkginfo["installer_item_location"] = relative_path
+        pkginfo["installer_item_location"] = uploaded_pkgpath.partition("/")[2]
 
         if self.env.get("uninstaller_pkg_path"):
-            relative_uninstall_path = self.copy_item_to_repo(
-                pkginfo, uninstaller_pkg=True
+            uploaded_uninstall_path = munkiimportlib.copy_item_to_repo(
+                repo,
+                self.env["uninstaller_pkg_path"],
+                pkginfo.get("version"),
+                self.env.get("repo_subdirectory", ""),
             )
-            pkginfo["uninstaller_item_location"] = relative_uninstall_path
+            pkginfo["uninstaller_item_location"] = uploaded_uninstall_path.partition(
+                "/"
+            )[2]
             pkginfo["uninstallable"] = True
```

Comment:
You're calling the exact same code twice with only one different argument - the installer path vs. potential uninstaller path. Rather than have literal duplicate copy-and-pasted code, factor this out into a function that takes the paths and updates the correct key, and then call the function for the uninstaller if it's present.